apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -123,16 +124,19 @@ export default class ModelManagerDialog extends React.Component {
   };
 
   deleteModel = () => {
+    this.setState({isDeletePending: true});
     $.ajax({
       url: `/api/v1/ml_models/${this.state.selectedModel.id}`,
       method: 'DELETE'
     }).then(response => {
       if (response.status === 'failure') {
         this.setState({
-          deletionStatus: `Model with id ${response.id} could not be deleted.`
+          deletionStatus: `Model with id ${response.id} could not be deleted.`,
+          isDeletePending: false
         });
       } else {
-        this.setState({confirmDialogOpen: false});
+        this.setState({confirmDialogOpen: false, isDeletePending: false});
+        this.getModelList();
```

Comment:
Should we show a spinner while `isModelListPending` is `true` as well?  (If not, should we keep isDeletePending` as `true` until the model list reload completes? 

(In theory we could avoid a fresh retrieval of models by just adjusting our local list, but I actually like the idea here of getting a fresh set of information.)